fastly/resource.go

```diff
@@ -157,7 +157,7 @@ func (c *Client) GetResource(i *GetResourceInput) (*Resource, error) {
 
 // UpdateResourceInput is used as input to the UpdateResource function.
 type UpdateResourceInput struct {
-	// Name is the name of the resource being linked to (e.g. an object store).
+	// Name is the name of the resource being linked to (e.g. an kv store).
```

Comment:
"an" is used before a word starting with a vowel, so with this change we should use "a" instead...
```suggestion
	// Name is the name of the resource being linked to (e.g. a kv store).
```